apiserver/plane/db/models/issue.py

```
@@ -15,6 +15,7 @@
 from plane.utils.html_processor import strip_tags
 from plane.db.mixins import SoftDeletionManager
 from plane.utils.exception_logger import log_exception
+from .base import BaseModel
```

Ruff (0.8.2)

`.base.BaseModel` imported but unused

Remove unused import: `.base.BaseModel`

(F401)